boardgamegeek/__init__.py

```diff
@@ -8,13 +8,15 @@
 """
 
 from .api import BGGClient, BGGChoose, BGGRestrictDomainTo, BGGRestrictPlaysTo, BGGRestrictSearchResultsTo, BGGRestrictCollectionTo
+from .legacy_api import BGGClientLegacy
 from .exceptions import BGGError, BGGApiRetryError, BGGApiError, BGGApiTimeoutError, BGGValueError, BGGItemNotFoundError
 from .cache import CacheBackendNone, CacheBackendMemory, CacheBackendSqlite
 from .version import __version__
 
 __all__ = ["BGGClient", "BGGChoose", "BGGRestrictSearchResultsTo", "BGGRestrictPlaysTo", "BGGRestrictDomainTo",
            "BGGRestrictCollectionTo", "BGGError", "BGGValueError", "BGGApiRetryError", "BGGApiError",
            "BGGApiTimeoutError", "BGGItemNotFoundError", "CacheBackendNone", "CacheBackendSqlite", "CacheBackendMemory"]
+__all__.append(["BGGClientLegacy"])
```

Comment:
I believe this should be `__all__.append("BGGClientLegacy")`

Reply:
Fixed in 27bc3fc887092bd01b7ecbf69ce6b86e92287e20